core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/CodeSystemVersionProperties.java

```diff
@@ -47,6 +45,6 @@
 	 * @return the effective date of this code system version (can be {@code null})
 	 */
 	@JsonFormat(shape=Shape.STRING, pattern="yyyyMMdd")
```

Comment:
Does this annotation do anything with a `String` typed property?

Reply:
Yes, I'm currently testing this, will let you know once I have something, I hope it does, if not, I'll remove it and add a default value to the API. Also, the `LocalDate` type could also work on the REST API.

Reply:
Cool, it renders the same example model in Swagger:

```
{
  "description": "string",
  "effectiveDate": "yyyyMMdd",
  "version": "string"
}
```